internal/integration/docker-compose.yaml

```diff
@@ -120,4 +120,13 @@ services:
     healthcheck:
       test: mysqladmin ping -ppass
     ports:
-      - 4308:3306
\ No newline at end of file
+      - 4308:3306
+
+# Default DB test, No Password
+  tidb:
+    platform: linux/amd64
+    image: pingcap/tidb:v5.4.0
```

Comment:
can't we use latest?

Reply:
We can but then if they change something in a later version (say v6.0.0) and our tests will break that's an unwanted behaviour.  
Maybe let's do both `tidb5:v5.4.0` and also a "rolling" `tidb_latest:latest` and when a new version breaks our tests we can add fixes to that version and then add `tidb<fail_version>:<fail_version>`

Reply:
>We can but then if they change something in a later version (say v6.0.0) and our tests will break that's an unwanted behaviour.

As I see it, that's a wanted behavior and helped me catch bugs in new versions of MySQL, PostgreSQL, and actually SQLite. Does pingcap maintain multiple versions or recommend working with latest?

Reply:
I see now that you added latest. nvm